templates/doi/cards/table-card.jsx

```diff
@@ -24,7 +26,12 @@ const formatDOI = (entity) => {
 const TableCard = ({ pages }) => (
   <Card className={styles.doiTableCard} tag="section">
     <Card.Title tag="h2">Browse DOI records</Card.Title>
-    <Table pages={pages} className={styles.doiTable} searchable>
+    <Table
+      pages={pages}
+      className={styles.doiTable}
+      excludeFooter={pages.data.length === 0}
```

Comment:
This is a little bit tricky. Data are "temporarily" nulled when user try to search. So it may cause a flickering effect.

In https://github.com/oacore/dashboard/pull/326 I removed `isFirstPage` flag and determine it out of data. If data is null it means data is loading if data === [] it means first page is loaded.

Reply:
I am not sure, I wanted to rely on data absence. If data is emptied on search, it's fine, if it's nulled — it should be emptied.

The problem is that we should disable search and sorting if there is an error.

Reply:
Disabling search should be a problem. Sorting may be a bit problematic. I think a new functional table supports change in children props. I need to check it more. Class table computed sorting configuration in the constructor. Another possibility is to disable sorting globally by some flag.

Reply:
As you said, disabling search was not a problem. I did not manage to work out with sorting so I will wait for #326.

@oacore/api we will need to disable sorting and filtering on backend to achieve data consistency in the 402 case.